### PR TITLE
Added Expeditor configs to check habitat build

### PR DESCRIPTION
**Pull request by saghoshprogress**
### Description

Added expeditor configs to check the habitat build.
We need to test the build for habitat for this build https://github.com/chef/supermarket/pull/2453

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -42,9 +45,11 @@ changelog:
     - "Type: Tech Debt": "Maintenance Updates"
 
 pipelines:
-  - omnibus/release:
-      env:
-        - IGNORE_CACHE: true # caching causes constant build failures
```
Comment (jeremiahsnapp) [inline .expeditor/config.yml:47]: are you sure you want to remove IGNORE_CACHE=true from the omnibus/release pipeline?

**Review by jeremiahsnapp — APPROVED** [review]
Nice work!

**Review by msys-sgarg — COMMENTED** [review]
Please add new lines to the end of the files, also I dont think removing IGNORE_CACHE is a good idea from the pipeline configurations.

File: .expeditor/config.yml (hunk 1 above)
Comment (tas50) [reply .expeditor/config.yml:47]: You really want this

**Comment by dheerajd-msys** [comment]
DCO sing-off is missing

**Comment by antima-gupta** [comment]
@saghoshprogress DCO sing-off is missing.

**Review by msys-sgarg — APPROVED** [review]
Looks good to me

**Review by dheerajd-msys — APPROVED** [review]

**Pull request merged**